### PR TITLE
pixel perfect 4x

### PR DESCRIPTION
**Pull request by aaaa1023**
## About The Pull Request
Adds pixel perfect 4x and 4.5x scaling. I tested it and it seems to work though 4.5 scaling  causes very slightly distorted pixel sizes. There's also a bit of letterboxing but I think that happens with all of the pixel perfect settings.

## Why It's Good For The Game
better fits 4k monitors and isn't stretch to fit 

## Changelog
:cl:
qol: adds pixel perfect 4x, 4.5x, and 5x
/:cl:

### CODE REVIEW

**Review by LemonInTheDark — APPROVED** [review]

File: tgui/packages/tgui/interfaces/PreferencesMenu/preferences/features/game_preferences/pixel_size.tsx [hunk 1]
```diff
@@ -9,5 +9,7 @@ export const pixel_size: Feature<number> = {
     1.5: 'Pixel Perfect 1.5x',
     2: 'Pixel Perfect 2x',
     3: 'Pixel Perfect 3x',
+    4: 'Pixel Perfect 4x',
+    4.5: 'Pixel Perfect 4.5x',
```
Comment (LemonInTheDark) [inline tgui/packages/tgui/interfaces/PreferencesMenu/preferences/features/game_preferences/pixel_size.tsx:13]: there is no reason this couldn't be an arbitrary input or a slider for the record

File: code/modules/client/preferences/pixel_size.dm [hunk 2]
```diff
@@ -4,7 +4,7 @@
 	savefile_identifier = PREFERENCE_PLAYER
 
 	minimum = 0
-	maximum = 3
+	maximum = 5
```
Comment (Mothblocks) [inline code/modules/client/preferences/pixel_size.dm:7]: You set the maximum to 5, yet there is no option for pixel perfect 5x

**Review by Mothblocks — CHANGES_REQUESTED** [review]

**Comment by JohnFulpWillard** [comment]
If you're also adding 5x can you add that to the changelog, and also is it fine in-game?

**Comment by aaaa1023** [comment]
> If you're also adding 5x can you add that to the changelog, and also is it fine in-game?

5x is too big for 2160p but probably the right size for 2880p. Also Changelog updated.

**Pull request merged**